hoomd/mpcd/force.py

```diff
@@ -26,30 +26,29 @@
 from hoomd.operation import _HOOMDBaseObject
 
 
-class SolventForce(_HOOMDBaseObject):
-    """Solvent force.
+class BodyForce(_HOOMDBaseObject):
```

Comment:
Would it be more descriptive to name this class `MPCDParticleForce`? I don't understand why they are called body forces if they are just forces on the MPCD particles.

Reply:
The term "body force" is used throughout the documentation also.

Reply:
I decided I would like to stick with `BodyForce`. Its closest analog is `hoomd.md.external.field.Field`, but these also aren't exactly fields because they specify a force rather than a potential